appinfo/info.xml

```diff
@@ -29,7 +29,7 @@ The rating depends on the installed text processing backend. See [the rating ove
 
 Learn more about the Nextcloud Ethical AI Rating [in our blog](https://nextcloud.com/blog/nextcloud-ethical-ai-rating/).
 	]]></description>
-	<version>3.7.0-alpha.2</version>
+	<version>3.7.0-alpha.3</version>
```

Comment:
3.7.0 has already been and gone - need to update to 3.8.0-alpha.x

Reply:
Oopsie, could you please fix it when squashing? Thanks :)